topi/python/topi/transform.py

```diff
@@ -129,6 +129,32 @@ def flip(a, axis=0):
     """
     return cpp.flip(a, axis)
 
+@tvm.tag_scope(tag=tag.INJECTIVE)
+def strided_slice(a, begin, end, stride=None):
+    """Slice of an array.
+
+    Parameters
+    ----------
+    a : tvm.Tensor
+        The tensor to be expanded.
+
+    begin: list of int
+        The indices to begin with in the slicing.
+
+    end: list of int
+        Indicies indicating end of the slice.
+
+    stride: list of int
```

Comment:
make stride optional, and rename it to slice

Reply:
@tqchen, thanks for your comment, in python if i rename the strided_slice to slice, i get a warning message in lint, as it clashes with built-in python slice() api, so need your opinion on this whether i should proceed on renaming to slice or keep the name same as strided_slice